index.html

```diff
@@ -164,13 +166,13 @@ <h2>Glossary</h2>
 
 <p><dfn data-lt="basic language range|Basic language range" class="lint-ignore export">Basic language range</dfn>. A [=language range=] consisting of a sequence of subtags separated by hyphens. That is, it is identical in appearance to a language tag.</p>
 
-<p><dfn class="lint-ignore export" data-lt="bidi isolate">Bidirectional isolate</dfn>. Defined by the [=Unicode Bidirectional Algorithm=] [[UAX9]], an <em>isolate</em> is a range of text whose contents do not participate in the embedding and reordering of the surrounding text the range appears in, which prevents the bidi algorithm from incorrectly ordering that content in relation to the surrounding text.</p>
-
-<p><dfn class="lint-ignore export">Bidi isolation</dfn>. The use of [=bidi isolates=] in text in order to prevent the automatic rules of the Unicode Bidirectional Algorithm incorrectly ordering that content in relation to the surrounding text. For example, numbers following right-to-left text in memory are automatically positioned to the left of that text by the Bidi Algorithm, but sometimes need to appear to the right. Another example occurs when lists of RTL items occur in a LTR sentence: the Bidi Algorithm will automatically assume that the order of items in the list should be "3 ,2 ,1", but actually what's needed is "1, 2, 3". In HTML, bidi isolation can be applied to a range of text by enclosing it in an element with a <code class="kw" translate="no">dir</code> attribute. In plain text there are Unicode formatting characters that can do the job. These mechanisms remove unwanted [=spillover effects=].</p>
+<p><dfn class="lint-ignore export" data-lt="bidi isolate">Bidirectional isolate</dfn>. A range of text, bounded by formatting characters or markup, that is treated by the [=Unicode Bidirectional Algorithm=] [[UAX9]] as directionally isolated from its surroundings. The entire range of text inside the isolate is treated by the surrounding text as if it were a single neutral character (such as <span class="codepoint" translate="no"><bdi lang="en">&#xFFFC;</bdi> [<span class="uname">U+FFFC OBJECT REPLACEMENT CHARACTER</span>])</span>, and is assigned the corresponding display position in the surrounding text. Furthermore, the text inside the isolate has no effect on the ordering of the text outside it, and vice versa.</p>
```

Comment:
This document is already in English, do we still need `lang=en` for the `bdi` here?

By the way, the font selected by my browser (`Menlo`) doesn't render anything for `U+FFFC`, but if I change the font to `Arial`, it will render an "OBJ" with a frame outside it, which I think is more intuitive. Alternatively, we can also mention only the Unicode code point and name for this kind of special character, not the actual character.

Reply:
Alternatively, you could replace the character itself with file:///Users/ishida/Sites/c/Specials/large/FFFC.png.  Put the character code in the alt text and people will still be able to copy paste it as characters.  You'll need to scale it by setting the height to 1.4rem, or something like that.

Reply:
I tried the graphic, but I'm getting funky results in Chrome. I think I'm going to leave this for now.